static/js/emoji_picker.js

```diff
@@ -69,13 +69,15 @@ function generate_emoji_picker_content(id) {
     if (id !== undefined) {
         var emojis_used = reactions.get_emojis_used_by_user_for_message_id(id);
         _.each(emojis_used, function (emoji_name) {
-            emojis[emoji_name] = {
-                name: emoji_name,
-                has_reacted: true,
-                css_class: emoji.emojis_name_to_css_class[emoji_name],
-                is_realm_emoji: emojis[emoji_name].is_realm_emoji,
-                url: emojis[emoji_name].url,
-            };
+            if (emoji.active_realm_emojis.hasOwnProperty(emoji_name)) {
+                emojis[emoji_name] = {
+                    name: emoji_name,
+                    has_reacted: true,
+                    css_class: emoji.emojis_name_to_css_class[emoji_name],
+                    is_realm_emoji: emojis[emoji_name].is_realm_emoji,
+                    url: emojis[emoji_name].url,
+                };
+            }
```

Comment:
>I don't understand this change; it would seem to me that it'll remove all the non-realm-emoji from this list?

This clearly seems to be a mistake on my side, will fix it!

>As a sidenote, do you think we want to show in the picker any realm emoji that you've reacted to already, even if they are deactivated (so that you can unreact)?

I think it will be good to not include deactivated realm emojis in the emoji picker since it can lead to confusion in case of multiple emojis with same name. If an user wants to unreact to an deactivated they have the option of doing so directly via clicking on that realm emoji which seems to be much more cleaner to me.